src/api/ZoweExplorerApi.ts

```diff
@@ -488,5 +488,13 @@ export namespace ZoweExplorerApi {
          * @returns the registered API instance
          */
         getJesApi(profile: IProfileLoaded): IJes;
+
```

Comment:
This method should be part of the ZoweExplorerExtender class along with the getProfile and getLinkedProfile methods. https://github.com/zowe/vscode-extension-for-zowe/pull/705